src/components/books.js

```diff
@@ -1,32 +1,19 @@
-import './books.css';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import Book from './book';
+import Form from './form';
 
-const Books = () => (
-  <div className="Lesson-Panel">
-    <div className="book-info">
-      <span className="School-of">Action</span>
-      <h2 className="Title">The Hunger Games</h2>
-      <span className="author">Suzanne-Collins</span>
-      <div className="options">
-        <span className="actions">Comments</span>
-        <span className="actions">Remove</span>
-        <span className="actions">Edit</span>
-      </div>
-    </div>
-    <div className="percent-graphic">
-      <div className="Oval-2" />
-      <div className="percent">
-        <span className="-Percent-Complete">64%</span>
-        <span className="Completed">Completed</span>
-      </div>
-    </div>
-    <div className="chapter">
-      <span className="Current-Chapter">CURRENT CHAPTER</span>
-      <span className="Current-Lesson">Chapter 17</span>
-      <div className="Rectangle-2">
-        <span className="Update-progress">UPDATE PROGRESS</span>
-      </div>
+const Books = () => {
+  const books = useSelector((state) => state.booksReducer);
+
+  return (
+    <div>
+      {books.map((book) => (
+        <Book key={book.id} book={book} />
+      ))}
+      <Form />
```

Comment:
- [ ] As the former code reviewer pointed out, the form is not rendering, so it is not possible for us to add and remove books at the moment, please kindly fix the issue. Please make this two functionalities(`add and remove` books) are implemented

![form](https://user-images.githubusercontent.com/65852621/151490342-405cc6e3-b4d0-455f-99f1-2938cc779055.png)
.

Reply:
Hi, @Mihndim2020. I'm concerned that you say that the app is not rendering. I tested in Chrome and Firefox and it's working. Could you give more details of the problem, as I'm not able to reproduce it and, in consequence, to fix it.

Reply:
Hello @Maclenn77, the app is actually rendering now, it wasn't rendering before, sure you were able to trace the issue.

![rendering](https://user-images.githubusercontent.com/65852621/151497479-7d223713-2812-4eca-8a86-285884b5b0e1.png)